src/components/menu-bar/project-title-input.jsx

```diff
@@ -19,47 +19,37 @@ const messages = defineMessages({
     }
 });
 
-class ProjectTitleInput extends React.Component {
```

Comment:
I should change this back to how it was before

Reply:
I think this is fine to do since your handler is coming from Redux now. I just think the name of the prop should change.

Reply:
Doing this...

Reply:
Done